build/cmake/ThriftMacros.cmake

```diff
@@ -17,6 +17,13 @@
 # under the License.
 #
 
+macro(ADD_PKGCONFIG_THRIFT name)
```

Comment:
Could we make this a function so that it can use the prefix, exec_prefix, libdir, and includedir variables without bleed-through?

Reply:
That seems to work. I'm not quite sure how to verify what is different between the two in this case.